packages/frontend/core/src/components/blocksuite/block-suite-editor/specs/custom/widgets/toolbar.ts

```diff
@@ -0,0 +1,112 @@
+import { notify } from '@affine/component';
+import { getAffineCloudBaseUrl } from '@affine/core/modules/cloud/services/fetch';
+import { I18n } from '@affine/i18n';
+import type { MoreMenuItemGroup } from '@blocksuite/affine-components/toolbar';
+import type { MoreMenuContext } from '@blocksuite/blocks';
+import { LinkIcon } from '@blocksuite/icons/lit';
+import {
+  DocsService,
+  type FrameworkProvider,
+  WorkspaceService,
+} from '@toeverything/infra';
+
+export function createToolbarMoreMenuConfig(framework: FrameworkProvider) {
+  return {
+    configure: <T extends MoreMenuContext>(groups: MoreMenuItemGroup<T>[]) => {
+      const clipboardGroup = groups.find(group => group.type === 'clipboard');
+
+      if (clipboardGroup) {
+        const copyIndex = clipboardGroup.items.findIndex(
```

Comment:
I prefer this kind of API to reduce the dependence on other items

```
// define in blocksuite
enum MoreMenuAnchor {
  Group = 1,
  ... = 2,
  ... = 3,
  Clipboard = 5
}

// define in affine
{
  config: {
    moreMenu: [
         {
            at: MoreMenuAnchor.Clipboard,
            icon: ...,
            type: ...,
            label: ....,
         }
    ]
  }
}

// render in blocksuite
moreMenuItems.sort((a,b) => a - b)
```

Reply:
Good idea. 

The current implementation is completely open, simple and crude.

btw, I might need to add an `op` type to allow adding, deleting, updating, etc.